.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Package",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/main.go",
+            "args": [
+                "start",
+                "--config",
+                "omnitruck.yml"
+            ],
+            "env": {
+                "CONFIG": "arn:aws:secretsmanager:us-east-1:056914312429:secret:omnitruck-dev-config-NiDaz3",
```

Comment:
Can we replace the config value here with a place-holder value?

Reply:
Done